CHANGELOG.md

```diff
@@ -1,5 +1,29 @@
 # Changelog
 
+## v0.2.0
+
+Breaking Changes:
+
+* Applicable release options in `mix.exs` are now scoped to a `:bakeware` key.
+  If you were setting `:compression_level` previously, you will need to update
+  you setting.
+
+Enhancements:
+
+* Windows Support! (Thanks @kritarthh :tada:) - See [Building on Windows](https://hexdocs.pm/bakeware/readme.html#building-on-windows)
+  for more info
+* Adds the `:start_command` option to the bakeware `mix` release options
+  which allows you to start the bakeware executable with the same comamnds
+  supported by `Mix.Release`. You can also specify this with the `--bw-command`
+  option when running the executable. See [`mix release` command options](https://hexdocs.pm/mix/Mix.Tasks.Release.html#module-bin-release_name-commands).
+* Instructions added for compiling static OpenSSL when needed (thanks @vans163)
+
+Bug Fixes:
+
+* Removes `mix bakeware.assemble` task which is unused and broken (Thanks @christhekeele).
+  Please use `mix release` for the same effect
+* Fix line editing when starting IEx via bakeware
```

Comment:
```suggestion
```
I reworked the feature add above to talk about this since the fix was to run `start_iex` rather than `start` on the release.